packages/devextreme-react/src/core/use-option-scanning.ts

```diff
@@ -75,12 +78,11 @@ export function useOptionScanning(
   };
 
   useLayoutEffect(() => {
-    if (!templateContainer) {
-      return;
-    }
-
-    const hasTemplateRendered = templateContainer.childNodes.length > 0;
+    const container = templateContainer && 'current' in templateContainer
```

Comment:
Minor - maybe use optional chaining here, smth like
const container = templateContainer?.current || templateContainer

Reply:
`templateContainer?.current` throws a TypeScript error, so `'current' in templateContainer` looks like _the_ valid way to narrow the type here